client/app/telecom/dashboard/bills/telecom-dashboard-bills.controller.js

```diff
@@ -6,22 +6,16 @@ angular.module("managerApp").controller("TelecomDashboardBillsCtrl", function (O
     self.links = {
         billing: REDIRECT_URLS.billing
     };
-
-    self.loaders = {
-        bills: true
-    };
-
     self.amountBillsDisplayed = 6;
-    self.sortby = "date";
-    self.reverse = true;
 
     /*= ================================
     =            API CALLS            =
     =================================*/
 
     function getLastBills () {
-        return OvhApiMeBill.Aapi().last().$promise.then(function (lastBills) {
-            self.last = lastBills.slice(Math.max(lastBills.length - self.amountBillsDisplayed, 0));
+        return OvhApiMeBill.Aapi().last().$promise.then(function (bills) {
+            const lengtToDisplay = Math.max(bills.length - self.amountBillsDisplayed, 0);
+            self.lastBills = _.sortBy(bills, "date").reverse().slice(lengtToDisplay);
```

Comment:
I think you doesn't need to sort anymore. The datagrid do it since you have set `data-sortable="desc"`.